### DISCUSSION TITLE
Running dev node with enabled cancun hard fork

### DISCUSSION THREAD

**Discussion by Deniallugo** [paradigmxyz/reth#7096] [Help] [answered]
By default if you run only `--dev` mode cancun is disabled, but even if change the code slightly and add cancun to the default setting  it returns: `Bad block with hash` 

If `--debug.tip` and  `--dev` enabled it just hangs on `Fetching tip block from the network.`

Is it possible to run the node with eip4844 txs and devnet?

**Comment by jsvisa** [comment, marked as answer]
https://github.com/paradigmxyz/reth/pull/7033 will support the cancun hard-fork

**Reply by Deniallugo** [reply]
Oh! fantastic! 
I hope [mattsse](https://github.com/mattsse) will have a look soon
